package.json

```diff
@@ -62,37 +62,6 @@
     "nodemailer-html-to-text": "latest",
     "nodemailer-sendmail-transport": "latest",
     "nodemailer-stub-transport": "latest",
-    "oae-activity": "",
```

Comment:
These are taken out in the node 6 PR, I'm not sure if we need to do it here as well... Although we should keep `"oae-rest": "latest"`?